src/components/CountrySelector/index.tsx

```diff
@@ -190,8 +195,8 @@ export abstract class CountrySelectionBase extends Component<Props, State> {
               onConfirm={this.handleCountrySearchConfirm}
             />
           </div>
-          {!this.isDocumentPreselected() &&
-            this.state.showNoResultsError &&
+          {((!this.isDocumentPreselected() && this.state.showNoResultsError) ||
+            this.state.alwaysShowEmptyMessage) &&
             this.renderNoResultsMessage()}
```

Comment:
This part could be written a bit more clearly/safe without all the `&&` and `||`.